packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -24,7 +24,8 @@ import {
 } from '@opentelemetry/core';
 import {ExporterOptions} from './external-types';
 import {GoogleAuth, JWT} from 'google-auth-library';
-import {google, monitoring_v3} from 'googleapis';
+import {monitoring} from 'googleapis/build/src/apis/monitoring';
+import type {monitoring_v3} from 'googleapis/build/src/apis/monitoring/v3';
```

Comment:
I played around with ncc and this PR. It looks the only import change needed is on L27. The `import type` statements didn't effect the bundle size–I guess `import type` is correctly getting stripped out since it is not needed at runtime.

Can you revert the `import type` lines in your PR and verify the size savings are the same still?
```suggestion
import type {monitoring_v3} from 'googleapis';
```